test/unit/test_keyring_multi.py

```diff
@@ -94,7 +104,12 @@ def test_keyring_with_generator_but_no_children():
             wrapping_algorithm=WrappingAlgorithm.AES_256_GCM_IV12_TAG16_NO_PADDING,
         )
     )
-    assert isinstance(test_multi_keyring.generator, RawAESKeyring)
+    assert isinstance(test_multi_keyring.generator, Keyring)
+    assert test_multi_keyring.generator.key_namespace == _PROVIDER_ID
+    assert test_multi_keyring.generator.key_name == _KEY_ID
+    assert test_multi_keyring.generator._wrapping_key == _WRAPPING_KEY_AES
+    assert test_multi_keyring.generator._wrapping_algorithm == WrappingAlgorithm.AES_256_GCM_IV12_TAG16_NO_PADDING
+    assert test_multi_keyring.children == ()
```

Comment:
Hedging our bets just in case we for some reason decide to use some other container class in the future.

```python
assert not test_multi_keyring.children
```